src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -300,6 +330,13 @@ ol {
   }
 }
 
+.confirmed-msg {
+  display: flex;
+  .confirmed-icon, .confirmed-note {
+    display: block;
```

Comment:
Or `d-flex` and `d-block` ?

My rule of thumb is to use CSS if:
- it's not possible/clean to put classes inline
- the class applies to multiple (repeated) elements

No action required, just food for thought.

Reply:
I'll give it a try